src/PactNet/RequestBuilder.cs

```diff
@@ -390,8 +398,20 @@ internal RequestBuilder WithJsonBody(dynamic body, JsonSerializerSettings settin
         {
             string serialised = JsonConvert.SerializeObject(body, settings);
             return this.WithBody(serialised, contentType);
+        }
+
+        /// <summary>
+        /// Set a body which is multipart/form-data but contains only one part, which is a file upload
```

Comment:
Should this description be the same as the overloaded one above (I think this version is the better description).

I'm also assuming the case of the parameter descriptions should have the first word capitalised? (ditto above method)